oaknational/Oak-Web-Application
# feat/ Handle download pages for lessons that don't have any content you can download#1317

# Description

Frames here:
https://www.figma.com/file/hQoMOEvJQlDJZ0n2ehftVT/Beta?node-id=4758%3A278769&t=zh4jiKhEXxoC7yhM-0

- Download page for a lessons that has no downloads should tell a user there are no downloads
- Remove all form inputs from the page

## Issue(s)

Fixes #1317 

## How to test

1. Go to beta/teachers/key-stages/ks1/subjects/maths/units/addition-and-subtraction-d935/lessons/adding-and-subtracting-with-5-and-3-and-with-6-and-3-6rv68t/downloads
3. You should see no download message

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
